### ISSUE TITLE
Test failures for 0.2.0

### ISSUE THREAD

**Issue by orhun** [ikebastuz/wiper#11]
Hey, AUR maintainer of `wiper` here 👋🏼 

I got the following test failures (in a clean Arch Linux chroot) while running `cargo check`:

```
failures:

---- delete::deleting_file_updates_parent_folders_sizes stdout ----
thread 'delete::deleting_file_updates_parent_folders_sizes' panicked at tests/delete.rs:259:9:
assertion `left == right` failed
  left: 4326
 right: 4462
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

---- delete::updated_current_folder_size stdout ----
thread 'delete::updated_current_folder_size' panicked at tests/delete.rs:231:9:
assertion `left == right` failed
  left: 4326
 right: 4462

---- delete::deleting_folder_updates_parent_folders_sizes stdout ----
thread 'delete::deleting_folder_updates_parent_folders_sizes' panicked at tests/delete.rs:336:9:
assertion `left == right` failed
  left: 4326
 right: 4462


failures:
    delete::deleting_file_updates_parent_folders_sizes
    delete::deleting_folder_updates_parent_folders_sizes
    delete::updated_current_folder_size

test result: FAILED. 11 passed; 3 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.11s
```

I'm not sure why they fail. As a workaround I am currently `--skip`ping them.

**Comment by ikebastuz** [comment]
Oh 😅 
Thank you for raising it!

I have an assumption, but not too sure. My bet is - empty folders do have metadata for 64 bytes on my machine. And I hardcoded it for tests. Can it be different for AUR?

Also - during debugging I noticed tiny inaccuracy in sizes. Will try to figure it out and let you know when fixed

Ofc - tests are working fine on my laptop :)
<img width="814" alt="image" src="https://github.com/ikebastuz/wiper/assets/24222413/ff62af68-d10f-45ae-af8c-9386dd3e4d5a">
